CHANGELOG.md

```diff
@@ -6,454 +6,494 @@ All notable changes to this project will be documented in this file.
+- CodeMirror version updated (used for text/yaml editors)
+- Default Filters tree converted to TreeBuilder - more on the way
+- Cloud Key Pair form converted to AngularJS (Dana - UX team)
+- Toolbars:Cleaned up partials, YAML -> classes
+- Provider Forms: Credentials Validation improvements
+- Updated PatternFly to v3.11.0
+- Summary Screen styling updates
 
 ## Removed
 
-  - Platform
-    - Removed rubyrep
-    - Removed hourly checking of log growth and rotation if > 1gb
-  - User Interface: Explorer Presenter RJS removal
-  - REST API: gems/cfme_client removed
+- Platform
+  - Removed rubyrep
+  - Removed hourly checking of log growth and rotation if > 1gb
+- User Interface: Explorer Presenter RJS removal
+- REST API: gems/cfme_client removed
 
 ## Fixed
 
 Notable fixes include:
-- Automate
-  - Fixed problem with request_pending email method
-  - Set User.current_user in Automation Engine to fix issue where provisioning resources were not being assigned to the correct tenant
-  - Provisioning: VMware Infrastructure: sysprep_product_id field is no longer required
-- Platform
-  - Use correct adjustment in chargeback reports
-  -  Replication: Fix typo prevention full error message
-  - Tenancy: User friendly tenant names
-- Providers
-  - Microsoft Azure: Fix proxy for template lookups
-  - VMware vSphere: Block duplicate events
-  - Hawkular: Fixes for LiveMetrics
-  - VMware: Fix for adding multiple disks
-  - Openstack: Catch unauthorized exception in refresh
-  - Middleware: Fix operation timeout parameter fetch
-  - Red Hat Enterprise Virtualization: Access VM Cluster relationship correctly
-- REST API
-  - Hide internal Tenant Groups from /api/groups
-  - Raise 403 Forbidden for deleting read-only groups
-  - API Request logging
-  - API: Fix creation of Foreman provider
-  - Ensure api config references only valid miq_product_features
-- SmartState: Update logging and job error message when getting the service account for Containers
-- User Interface
-  - Add missing Searchbar and Advanced Search button
-  -  Containers: Download to PDF/CSV/Text - don't download deleted containers
-  -  Allow bring VM out of retirement from detail page
-  -  Inconsitent menues in Middleware Views
-  - Save Authentication status on a Save
-  - User friendly tenant names
-  -  RBAC:List only those VMs that the user has access to in planning
+
+### Automate
+
+- Fixed problem with request_pending email method
+- Set User.current_user in Automation Engine to fix issue where provisioning resources were not being assigned to the correct tenant
+- Provisioning: VMware Infrastructure: sysprep_product_id field is no longer required
+
+### Platform
+
+- Use correct adjustment in chargeback reports
+- Replication: Fix typo prevention full error message
+- Tenancy: User friendly tenant names
+
+### Providers
+
+- Microsoft Azure: Fix proxy for template lookups
+- VMware vSphere: Block duplicate events
+- Hawkular: Fixes for LiveMetrics
```

Comment:
I believe LiveMetrics is new for Euwe, so this is not necessary.  cc @abonas am I right here?